source/framework/vx_tensor.c

```diff
@@ -342,6 +343,14 @@ VX_API_ENTRY vx_tensor VX_API_CALL vxCreateTensor(
             tensor = (vx_tensor)ownGetErrorObject((vx_context)context, status);
         }
     }
+    if (status != VX_SUCCESS)
+    {
+        if (tensor)
```

Comment:
MISRAC ... should check against NULL

Reply:
removed as the check is not necessary. The release tensor  and ownGetErrorObject functions are called before (if needed).

Reply:
It looks like this is still in place, perhaps the latest version has not been pushed?